### DISCUSSION TITLE
I'm not sure why create-next-app has permission issues

### DISCUSSION THREAD

**Discussion by ZCY-G** [vercel/next.js#63593] [Help]
### Summary

Create a project using `pnpm create next-app`. The prompt `Success! Created my-app at /Users/username/Documents/GitHub/my-app` is followed by `Aborting installation.`. The error message is `Unexpected error. Please report it as a bug:
  Error: EACCES: permission denied, open '/Users/username/Library/Preferences/create-next-app-nodejs/config.json.tmp-107400408351cdb6'`.
`which node`: `/Users/username/.nvm/versions/node/v20.11.1/bin/node`
`which pnpm`: `/Users/username/.nvm/versions/node/v20.11.1/bin/pnpm`

**Comment by aziyatali** [comment]
Apparently,  pnpm doesn't have permission to create file inside Documents folder.

Try in Desktop or inside `/Users/username`

**Reply by ZCY-G** [reply]
pnpm created the folder successfully and there are files in the folder.

**Reply by aziyatali** [reply]
I think my previous response was incorrect. You may create file or folder anywhere in the computer based on write permission in the operating system. What I wanted say is during package installation, some of them may fail to install inside Documents.
You can not install to the Documents by default and by design of the macOS.  Have you tried inside username folder?

**Reply by ZCY-G** [reply]
After my try, the result is the same. I think it is a permissions issue with the `create-next-app-nodejs` directory. When I changed the staff permissions to read and write, the error no longer occurred, but I think this is not a good solution.
